src/Ouroboros/Network/Protocol/ChainSync/Server.hs

```diff
@@ -0,0 +1,115 @@
+{-# LANGUAGE GADTs           #-}
+{-# LANGUAGE DataKinds       #-}
+{-# LANGUAGE NamedFieldPuns  #-}
+
+-- | A view of the chain synchronisation protocol from the point of view of the
+-- server.
+--
+-- This provides a view that uses less complex types and should be easier to
+-- use than the underlying typed protocol itself.
```

Comment:
This comments seems not right, I guess copy pasted from `Type.hs`.

Reply:
No this isn't a copy&paste'o. It's the same comment for the client and server side, but that's true in both cases.